0002/README.md

```diff
@@ -0,0 +1,197 @@
+
+## Goals and non-Goals
+
+Goals are:
+
+1. Define what a release is and what it is comprised of a release
+2. Define standard policies and practices for releases that all Tinkerbell
+   projects should follow
+3. Define Quality Assurance polices for releases.
+
+Non-Goals:
+
+1. How each project should be released and what it should release
+2. Define who own releases
+3. Distribution via `apt`, `apk` and so on
+4. Define a roadmap and how it should be presented to the community.
+5. Support and deprecation lifecycle
+
+## Content
+
+A release is a tested and stable point in time snapshot from a codebase.
+
+We follow the [Semantic Versioning (semver) 2.0.0](https://semver.org).
+Summarized as follows:
+
+Given a version number MAJOR.MINOR.PATCH, increment the:
+
+* MAJOR version when you make incompatible API changes,
+* MINOR version when you add functionality in a backwards compatible manner, and
+* PATCH version when you make backwards compatible bug fixes.
+
+Additional labels for pre-release and build metadata are available as extensions
+to the MAJOR.MINOR.PATCH format.
+
+### Schedule
+
+Every subproject should be able to follow its own cadence in terms of how often
+or regularly a new release should be cut, as long as it is documented and
+shared with the community. It is a responsibility for the maintainer to figure
+out a good strategy that better fits their workload, process and codebase.
+
+A new `MINOR` version for Tinkerbell should be released following a proper
+schedule, right now I think every two months sounds reasonable looking at the
+code written and merged in the last couple of weeks. In this way the
+subprojects will have an intrinsic release target if they want to get their new
+code included in the Tinkerbell release.
+
+`PATCH` release for Tinkerbell can be done as required by the subprojects
+bug fixes.
+
+`MAJOR` release right now are not even evaluated because the project is too
+young. By definition breaking changes require a new `MAJOR` release, right now
+any of our projects have a release yet, we are working hard to reach `v1.0.0`,
+until there bc breaks are allowed but they will be notified with a proper label
+`bc-break` attached to the pull request.
+
+### GitHub release and release website
+
+Git supports tag and GitHub has the concept of a release based on a particular
+tag.
+
+The artifact generated as part of the release will be published as part of the
+release website, using as a guidance how [HashiCorp
+Release](https://releases.hashicorp.com/) works.
+
+I like the idea to use a release website vs having to push them as manifest in
+GitHub release because it gives a single place where the community can
+programmatically lookup what they are looking for, without having to look for
+repositories, versions and so on.
+
+As part of the Release page on GitHub we will publish a generated changelog.
+
+### Distribution
+
+Docker containers, binaries, packages such as `deb`, `rpm` requires
```

Comment:
We can publish debs to a PPM via launchpad.net (ubuntu/debian official path): https://help.launchpad.net/Packaging/PPA

I don't know about rpm build/archive sites, but a quick Google turns up: https://build.opensuse.org/ and https://fedoraproject.org/wiki/Koji

We could defer these types of archives for a later proposal (or we could treat them like simple binaries and stash them with each github release).